<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Saturation Change</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGraphicsView" name="originalPicture"/>
        </item>
        <item>
         <widget class="QGraphicsView" name="resultPicture"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="BeforeGroupBox">
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="satLvlLabel">
           <property name="geometry">
            <rect>
             <x>360</x>
             <y>200</y>
             <width>61</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0%</string>
           </property>
          </widget>
          <widget class="QLabel" name="cppTimeLabel">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>110</y>
             <width>171</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QSlider" name="saturationSlider">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>160</y>
             <width>160</width>
             <height>22</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>80</y>
             <width>131</width>
             <height>31</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ASM time:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>200</y>
             <width>171</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Saturation level: </string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>10</y>
             <width>101</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>18</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Before</string>
           </property>
          </widget>
          <widget class="QLabel" name="asmTimeLabel">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>80</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>110</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>C++ time:</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="AfterGroupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>250</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>10</y>
             <width>91</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>18</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>After</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>190</y>
             <width>81</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Threads: </string>
           </property>
          </widget>
          <widget class="QPushButton" name="runButton">
           <property name="geometry">
            <rect>
             <x>360</x>
             <y>80</y>
             <width>111</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
          <widget class="QPushButton" name="loadButton">
           <property name="geometry">
            <rect>
             <x>360</x>
             <y>160</y>
             <width>111</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="cppRadioButton">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>110</y>
             <width>89</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>C++</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="threadsNumLabel">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>190</y>
             <width>51</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="asmRadioButton">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>80</y>
             <width>89</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ASM</string>
           </property>
          </widget>
          <widget class="QSlider" name="threadsSlider">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>160</y>
             <width>121</width>
             <height>22</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>64</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
